proposals/0127-get-sysvar-syscall.md

```diff
@@ -155,13 +155,15 @@ The syscall aborts the virtual machine if any of these conditions are true:
 - `var_addr + length` is not in `[0, 2^64)`.
 - Compute budget is exceeded.
 
-The syscall returns `1` if the sysvar is not present in the Sysvar Cache.
+All VM violations (above) are checked first, before attempting to return
+gracefully.
```

Comment:
```suggestion
All VM violations (above) are checked first, before to returning one of the following error codes. The following checks are completed in the order they appear below.
```

Reply:
Note I flipped these around since we can't know the length of the sysvar data if it's not found.